src/SonarAnalyzerEmbedder/SonarAnalyzerEmbedder.CSharp.csproj

```diff
@@ -49,14 +49,12 @@
     <Reference Include="System" />
   </ItemGroup>
   <ItemGroup>
-    <None Include="packages.SonarAnalyzerEmbedder.CSharp.config">
-      <SubType>Designer</SubType>
-    </None>
+    <None Include="packages.SonarAnalyzerEmbedder.CSharp.config" />
```

Comment:
Is it a Visual Studio change?

Reply:
Yes. The <Designer> subtype is added when a file is opened using various types of editor. It has not effect on the build.